gadgets/fsnotify/test/unit/fsnotify_test.go

```diff
@@ -87,17 +89,31 @@ type testDef struct {
 }
 
 func TestFsnotifyGadget(t *testing.T) {
-	gadgettesting.MinimumKernelVersion(t, "5.10")
+	gadgettesting.MinimumKernelVersion(t, "5.4")
 	gadgettesting.InitUnitTest(t)
 	runnerConfig := &utilstest.RunnerConfig{}
 
+	currVersion, err := kernel.GetKernelVersion()
+	require.NoError(t, err, "Failed to parse kernel version: %s", err)
+	ignoreInvalidIno := false
+	if utils.CurrentTestComponent == utils.KubectlGadgetTestComponent {
```

Comment:
`MinimumKernelVersion` internally checks the kernel version of the deployed gadget nodes. Could you add maybe another function which would return you the version (Something like `GetKernelVersion` for us)

So this check could be more specific

Reply:
Done! PR updated.